src/sentry/api/endpoints/organization_events_root_cause_analysis.py

```diff
@@ -25,13 +23,24 @@
 BASE_REFERRER = "api.organization-events-root-cause-analysis"
 SPAN_ANALYSIS = "span"
 GEO_ANALYSIS = "geo"
+SPAN_ANALYSIS_SCORE_THRESHOLD = 1
```

Comment:
I'm not sure if this threshold is too high. I came across an issue where in the before/after group the duration never changed, but the TPM was slightly different and hence the score was very close to 0 (e.g. 0.00006...) but still positive and ended up in the result.

I think we could likely tune this down, or I could make it a parameter so the frontend can try different values.

Reply:
Let's start with a default of 0 and override it from the frontend